flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/FlinkOperator.java

```diff
@@ -165,7 +167,12 @@ void registerDeploymentController() {
         var metricManager =
                 MetricManager.createFlinkDeploymentMetricManager(baseConfig, metricGroup);
         var statusRecorder = StatusRecorder.create(client, metricManager, listeners);
-        var autoscaler = AutoscalerFactory.create(client, eventRecorder);
+        Duration refreshClusterViewInterval =
+                configManager
+                        .getDefaultConfig()
+                        .get(KubernetesOperatorConfigOptions.REFRESH_CLUSTER_RESOURCE_VIEW);
+        var clusterResourceManager = new ClusterResourceManager(refreshClusterViewInterval, client);
```

Comment:
We should add a method to create the `ClusterResourceManager` and we should probably have a config to disable it by default. This is only applicable in certain Kubernetes envs which are actually resource constrained only by node capacity and not by other quotas

Reply:
Yes, method created. Changed the default to be disabled. The feature can be extended as more contributors work on this.